### ISSUE TITLE
[A11y] Implment keyboard navigation within Annotation component

### ISSUE THREAD

**Issue by Dananji** [samvera-labs/ramp#781]
### Description

When MarkersDisplay component is used for annotations display, it can have a larger number of focus-able child elements as annotation rows. When using keyboard navigation the user cannot reach the annotation set drop-down menu or any annotation in the Annotations component.

To access annotation rows in the component, roving tab-index strategy can be used identical to the Transcript component.

Solution:
- Use `Tab` and `Shift`+`Tab` keys to navigate into and out of the component and between drop-down menu and annotations display container
- Use `Up` and `Down` arrow keys along with `Enter`/`Space` keys to reach and select annotations respectively

### Done Looks Like

Use ARIA-role and tab-index attributes as needed to; 
- [ ] implement roving tab-index strategy for navigating annotations within the annotations component
- [ ] add annotation-set drop-down menu to the tab order

**Comment by Dananji** [comment]
This implementation doesn't use the roving tabIndex strategy. With the list-based implementation of the annotation row display I was able to use `.focus()` on each annotation row to move focus as the user goes through the annotations in the component. 
In the transcript component, we needed the roving tabIndex strategy since the cue are displayed as buttons which inserts each cue into the tab-order of the DOM, while it doesn't happen with the list implementation in the annotations component.

Notes for testing keyboard navigation:
- Once the user tabs into the `Annotations` tab, pressing `Tab` they can move focus on the dropdown menu, auto-scroll checkbox, and annotations display in order
- When focus is on the annotation set select dropdown menu
  - pressing either `Enter`/`Space` key toggles the dropdown menu
  - pressing `ArrowDown`/`ArrowUp` keys opens the dropdown menu
  - once the dropdown menu is opened pressing `ArrowDown`/`ArrowUp` allows the user to focus on each annotation set option in the list
  - once focus is on an option pressing `Enter`/`Space` selects/un-selects the annotation set
  - once focus is on an option pressing `Escape`/`Tab` key closes the dropdown menu
- When focus is on the auto-scroll check box 
  - pressing `Enter`/`Space` checks/un-checks the auto-scroll
  - pressing `Tab` moves the focus to the first annotation row
- When focus is on an annotation row
  -  pressing `Enter`/`Space` keys selects the annotation
  -  pressing `ArrowDown`/`ArrowUp` keys moves the focus to the next/previous annotation respectively
  -  pressing `Shift`+`Tab` keys moves the focus away from annotation row to its parent container
  -  pressing `Tab` key when focused on an annotation with plain text does nothing
  -  pressing `Tab` key when focused on an annotation with links or show more/less button moves the focus into these links or buttons

**Comment by joncameron** [comment]
Works great! Closing this issue; the only issue I noticed in testing this is another div accidentally getting focus, and I created a new issue for that: https://github.com/samvera-labs/ramp/issues/845.
